docs/ycsb.md

```diff
@@ -16,7 +14,7 @@ Your resource file may look like this:
 apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
 kind: Benchmark
 metadata:
-  name: ycsb-couchbase-benchmark
+  name: ycsb-mongo-benchmark
```

Comment:
Should you rename this to `docs/ycsb-mongo.md` to be specific that this is for mongodb?

Reply:
Only the example here is specifically for running against mongodb, so I dont think so.

Reply:
The remove mongo?